mlc_llm/core.py

```diff
@@ -558,7 +571,8 @@ def build(mod_deploy: tvm.IRModule, args: argparse.Namespace) -> None:
     with tvm.transform.PassContext(config={"relax.backend.use_cuda_graph": use_cuda_graph}):
         # The num_input attribute is needed to capture transformed weights passed as input
         # into a cuda graph.
-        mod_deploy["decode"] = mod_deploy["decode"].with_attr({"num_input": 3})
+        if not args.enable_batching:
+            mod_deploy["decode"] = mod_deploy["decode"].with_attr({"num_input": 3})
```

Comment:
Is there any reason to disable cuda graph for batching? If so, good to document the reason.

Reply:
In batching the decode function is named `"decode_with_embed"` and only has two inputs. And the attention computation in batching is more different than the unbatched version. I have no experience with CUDA graph so am unable properly handle it from my knowledge. Just added a note to leave the CUDA graph support for batching as a TODO item.

Reply:
I suppose batching should be compatible with cuda graph and we can enable it later.